lib/cisco_node_utils/cmd_ref/span_session.yaml

```diff
@@ -0,0 +1,64 @@
+# SPAN session
```

Comment:
Curious about why you named this object `span_session` rather then `monitor_session`.